Sources/CodeGen/CXX/CXXTranspiler.swift

```diff
@@ -603,4 +600,63 @@ public struct CXXTranspiler {
 
   }
 
+  // MARK: entry point function body
+
+  /// Returns the transpiled body of the entry point function in `source`.
+  func entryPointBody(module source: ModuleDecl.Typed) -> CXXScopedBlock? {
+    let elements = Array(source.topLevelDecls)
+    return elements.first(transformedBy: { (d) in
+      entryPointBody(topLevel: d, in: source)
+    })
+  }
+  /// Returns the transpiled body of the entry point function that may be calling `source`.
```

Comment:
```suggestion
  /// Return the body of a function calling `source` if `source` is the entry point of `module`.
  /// Otherwise, returns `nil`.
```

The current docs suggests this function is unconditionally returning scoped block.

Reply:
not applicable anymore if I get rid of this function